docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
@will-lauer 

1. Is that what you mentioned "scala-based" implementation is "com.wandoulabs.avro:wandou-avpath"? It can be used by Java since scala will be compiled into JVM bytecode and can interoperate with Java. One thing worth to note is the performance of that library.
1. If we decided finnally support similar mechanisms in both avro and JSON and maybe something else, and the mechanisim will based on JSONPath, then maybe we can reuse the `JSONParseSpec` in AvroInputRowParser?